app/components/search-preprints.js

```diff
@@ -23,21 +23,22 @@ export default Component.extend({
     theme: service(),
     actions: {
         search() {
-            let query = $.trim(this.$('#searchBox').val());
+            const query = $.trim(this.$('#searchBox').val());
+            /* eslint-disable-next-line ember/closure-actions */
```

Comment:
it doesn't like send actions but haven't found a good way around it. Recommendations appreciated

Reply:
When pass it in [index.hbs](https://github.com/CenterForOpenScience/ember-osf-preprints/blob/release/next-interfaces/app/templates/index.hbs#L31) you should be able to change `search="search"` to `search=(action 'search')` and then change `this.sendAction('search', query);` to `this.get('search')(query)`.

For more info: https://alexdiliberto.com/posts/ember-closure-actions/